src/backend/web/routes/index.js

```diff
@@ -17,11 +17,19 @@ const query = require('./query');
 
 const router = express.Router();
 
+/**
+ * In staging and production, our revers proxy takes care of serving the content in the public folder.
+ * We're keeping this route for development.
+ */
 router.use(express.static(path.join(__dirname, '../../../frontend/public')));
```

Comment:
If this is only for development, should we wrap this in a `if(process.NODE_ENV !== 'production')` or something?

Reply:
I honestly don't know. In production and staging, this route will never be used, so I'm open to suggestions.

Reply:
In `env.staging` we have `NODE_ENV=production`, and `env.example` has `NODE_ENV=development` so let's do this:

```js
if(process.NODE_ENV === 'devleopment') {
  router.use(express.static(path.join(__dirname, '../../../frontend/public')));
}
```